tests/st/calicoctl/test_crud.py

```diff
@@ -1479,369 +1479,366 @@ def test_get(self):
+                                'selector': ""}}),
+                   ("policy-invalidAction", {
+                       'apiVersion': API_VERSION,
+                       'kind': 'NetworkPolicy',
+                       'metadata': {'name': 'policy2'},
+                       'spec': {'Egress': [{'action': 'jumpupanddown',  # invalid action
+                                            'destination': {},
+                                            'protocol': 'TCP',
+                                            'source': {},
+                                            }],
+                                'Ingress': [{'action': 'Allow',
+                                             'destination': {},
+                                             'protocol': 'UDP',
+                                             'source': {}}],
+                                'order': 100000,
+                                'selector': ""}}),
+                   ("policy-NetworkPolicyNameRejected", {
+                       'apiVersion': API_VERSION,
+                       'kind': 'NetworkPolicy',
+                       'metadata': {'name': 'knp.default.rejectmeplease',
+                                    'namespace': 'default'},
+                       'spec': {'Egress': [{'action': 'Allow',
+                                            'destination': {},
+                                            'protocol': 'TCP',
+                                            'source': {},
+                                            }],
+                                'Ingress': [{'action': 'Allow',
+                                             'destination': {},
+                                             'protocol': 'UDP',
+                                             'source': {}}],
+                                'order': 100000,
+                                'selector': ""}}),
+                   ("pool-invalidNet1", {'apiVersion': API_VERSION,
+                                         'kind': 'IPPool',
+                                         'metadata': {},
+                                         'spec': {'ipip': {'mode': 'Always'},
+                                                  'cidr': "10.0.1.0/33"}  # impossible mask
+                                         }),
+                   ("pool-invalidNet2", {'apiVersion': API_VERSION,
+                                         'kind': 'IPPool',
+                                         'metadata': {},
+                                         'spec': {'ipip': {'mode': 'Always'},
+                                                  'cidr': "10.0.256.0/24"}  # invalid octet
+                                         }),
+                   ("pool-invalidNet3", {'apiVersion': API_VERSION,
+                                         'kind': 'IPPool',
+                                         'metadata': {},
+                                         'spec': {'ipip': {'mode': 'Always'},
```

Comment:
This is no longer the format for the IPPool.

There is now a top-level ipipMode field, rather than this sub-struct.